.github/workflows/ci.yml

```diff
@@ -10,35 +10,44 @@ jobs:
   test-webpack4:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set node version to 16
-        uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
         with:
-          node-version: 16
-          cache: 'yarn'
-      - run: yarn install
-      - run: yarn test:webpack4
+          version: 8.12.0
```

Comment:
Let's add `packageManager` field into `package.json` and remove this.

Reply:
Same for below

Reply:
done